src/Site/views/shared/container/xforge.navbarApp.js

```diff
@@ -10,49 +10,34 @@ angular.module('xforge.navbarApp', ['bellows.services', 'ui.bootstrap', 'pascalp
     };
   }])
   */
-  .controller('navbarController', ['$scope', 'modalService', '$location', 'sessionService',
-  function ($scope, modalService, $location, ss) {
+  .controller('navbarController', ['$scope', 'modalService', '$location', 'asyncSession', '$window',
```

Comment:
Having two session services right now is purely for the sake of ease of porting dependencies to use an asynchronous session service. My plan has been to delete the sessionService and rename asyncSession to sessionService with a find and replace. I'm not planning to use asyncSession as a permanent name, just during work in progress while the sessionService name is already taken.